uninstall.php

```diff
@@ -20,6 +20,11 @@ class PLL_Uninstall {
 	public function __construct() {
 		global $wpdb;
 
+		// Don't do anything except if the constant PLL_REMOVE_ALL_DATA is explicitely defined and true.
+		if ( ! defined( 'PLL_REMOVE_ALL_DATA' ) || ! PLL_REMOVE_ALL_DATA ) {
+			return;
```

Comment:
See my previous comment.

Returning from a constructor outlines the fact that this class isn't used in an OOP way. I believe we should never perform any logic in a class constructor, as this makes them quite hard to instantiate in our automated tests.

Reply:
I did not want to rewrite everything. Just add this constant to control the uninstall. Although it has downsides for testing.